root/view/src/main/resources/view/fxml/user/UserView.fxml

```diff
@@ -8,9 +8,9 @@
 <?import javafx.scene.control.Label?>
 <?import javafx.scene.layout.Pane?>
 
-<BorderPane fx:id="_borderPane" stylesheets="@../css/BaseStile.css" xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1">
+<BorderPane fx:id="_borderPane" stylesheets="@../../css/BaseStile.css" xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1">
    <top>
-      <Label fx:id="_titelLabel" styleClass="label-header" stylesheets="@../css/BaseStile.css" text="Create New User" BorderPane.alignment="TOP_LEFT" />
+      <Label fx:id="_titelLabel" styleClass="label-header" stylesheets="@../../css/BaseStile.css" text="Create New User" BorderPane.alignment="TOP_LEFT" />
```

Comment:
afaik you only need to reference the stylesheet in the `BorderPane` and it will be available in all child controls